core/src/main/java/io/jenkins/pluginhealth/scoring/probes/GitHubOpenIssuesProbe.java

```diff
@@ -47,6 +47,10 @@ class GitHubOpenIssuesProbe extends AbstractOpenIssuesProbe {
      */
     @Override
     Optional<Integer> getCountOfOpenIssues(ProbeContext context) {
+        if (context.getIssueTrackerUrlsByNames().get("github") == null) {
```

Comment:
What I meant was to check that:

`if (context.getIssueTrackerUrlsByNames() == null)`